src/main/java/no/nav/sbl/util/AuthIntrospectionFilter.kt

```diff
@@ -26,7 +26,11 @@ class AuthIntrospectionFilter : Filter {
                 it.name
             }
         )
-        append("Referer: ")
-        appendLine(request.getHeader("Referer"))
+        append("Headers: ")
+        val allHeaders = mutableListOf<String>()
```

Comment:
Siden det bare er for debugging har det ikke så mye å si. 
Men har kunne man f.eks gjort noe det nedenfor , og dermed sluppet å lage en ny liste som må joines sammen etc.
```kotlin
        appendLine("Headers: ")
        for (headerName in request.headerNames) {
            appendLine("\t$headerName: ${request.getHeader(headerName)}")
        }
```